virtual_rainforest/models/plants/plants_model.py

```diff
@@ -68,6 +89,8 @@ class PlantsModel(BaseModel):
     vars_updated = (
         "leaf_area_index",  # NOTE - LAI is integrated into the full layer roles
         "layer_heights",  # NOTE - includes soil, canopy and above canopy heights
+        "layer_fapar",
+        "layer_absorbed_irradiation",
         "herbivory",
         "transpiration",
         "canopy_evaporation",
```

Comment:
Is the distinction here that `transpiration` is water from the canopy interception pool and `canopy_evaporation` is the water that plants such out of the soil? If so, I will update the input to the hydrology model to `canopy_evaporation` to account for root water uptake. Also, is this going to be an accumulated values over all layers?

Reply:
The other way around, isn't it? Transpiration should be the stomatal conductance rate scaled up to the amount of photosynthesis and `canopy_evaporation` is water loss from the (currently non existent) canopy interception pool. I've not implemented any of that though, so if there is something better to do, let me know! 

I can do transpiration per canopy layer - the stomatal conductance is a prediction of the P Model, which will be run within layers. In fact, I'll have to - but can always sum it if we don't need the vertical profile. For the evaporation - if we have a layer specific interception model, then that too! I think that's another PR though.

Reply:
yes, sorry, I mixed that up, I did mean `transpiration`.
I have implemented a simple interception pool for the whole canopy (then I got carried away and changed a million other little things and it's now a complete overhaul of the hydrology model, which you will soon see...), so we could include it in one of the future PRs in the hydrology model, I don't think it necessarily needs to touch the plant model because it only needs to LAI.

Reply:
It's also only relevant for us if we want to close the water balance